docs/intro.md

```diff
@@ -0,0 +1,38 @@
+# Contile
+
+Contile is a service that fetches tiles to support the Sponsored Tiles feature in Firefox. Sponsored Tiles are a set of display ads appearing on the Firefox New Tab page provided by either a paying partner, or a selected set of other partners.
+
+## Table of Contents
+- [api.md - API Documentation][1]
+- [setup.md - Developer Setup Documentation][2]
+
+[1]: ./api.md
+[2]: ./setup.md
+
+
+## Architecture
+
+```mermaid
```

Comment:
This is a fantastic diagram, it makes it easy to get a glimpse of the whole system! 🤩

Reply:
![Screenshot 2023-08-03 at 8 27 04 PM](https://github.com/mozilla-services/contile/assets/25379936/00971c52-6852-45ba-8864-975561aa20c7)
🙌 success

Reply:
OMG, my OCD thanks you!

Contile - a tiny piece that does all the heavy lifting. Works all night sleeps all day... 😀

Reply:
Looks lovely!